src/layouts/components/Header.tsx

```diff
@@ -327,43 +327,72 @@ function MenuItems(): JSX.Element {
 }
 
 function MoreDropdown(): JSX.Element {
+  const [isItemClicked, setIsItemClicked] = useState(false);
+
+  const router = useRouter();
   return (
     <Menu as="div" className="relative">
-      <Menu.Button className="flex flex-row items-center mx-4 dark:hover:text-dark-50 cursor-pointer text-lg hover:text-primary-500">
-        More
-        <IoChevronDown className="ml-2" size={20} />
-      </Menu.Button>
-      <Transition
-        enter="transition duration-100 ease-out"
-        enterFrom="transform scale-95 opacity-0"
-        enterTo="transform scale-100 opacity-100"
-        leave="transition duration-75 ease-out"
-        leaveFrom="transform scale-100 opacity-100"
-        leaveTo="transform scale-95 opacity-0"
-      >
-        <Menu.Items className="absolute m-4 min-w-max flex flex-col divide-y bg-white border rounded-lg border-gray-200 dark:border-gray-700 dark:bg-gray-700">
-          {[
-            {
-              name: "Consortium",
-              link: "/consortium/asset_breakdown",
-            },
-            {
-              name: "On Chain Governance",
-              link: "",
-            },
-          ].map((item, index) => {
-            return (
-              <Menu.Item key={index}>
-                <Link href={{ pathname: item.link }}>
-                  <a className="px-6 py-3.5 cursor-pointer text-sm border-gray-200 hover:text-primary-500 dark:hover:text-dark-50">
-                    {item.name}
-                  </a>
-                </Link>
-              </Menu.Item>
-            );
-          })}
-        </Menu.Items>
-      </Transition>
+      {({ open }) => (
+        <>
+          <Menu.Button
+            className={classNames(
+              "flex flex-row items-center mx-4 dark:hover:text-dark-50 cursor-pointer text-lg hover:text-primary-500",
+              {
+                "dark:text-dark-50 text-primary-500": isItemClicked,
+              }
+            )}
+          >
+            More
+            {open ? (
+              <IoChevronDown className="ml-2" size={20} />
+            ) : (
+              <IoChevronUp className="ml-2" size={20} />
+            )}
+          </Menu.Button>
+          <Transition
+            enter="transition duration-100 ease-out"
+            enterFrom="transform scale-95 opacity-0"
+            enterTo="transform scale-100 opacity-100"
+            leave="transition duration-75 ease-out"
+            leaveFrom="transform scale-100 opacity-100"
+            leaveTo="transform scale-95 opacity-0"
+          >
+            <Menu.Items className="absolute m-4 min-w-max flex flex-col divide-y bg-white border rounded-lg border-gray-200 dark:border-gray-700 dark:bg-gray-700">
+              {[
+                {
+                  name: "Consortium",
+                  link: "/consortium/asset_breakdown",
+                },
+              ].map((item, index) => {
+                if (router.pathname === item.link) {
```

Comment:
need to update logic, proof of assets is not covered for active state 🤔 
<img width="1052" alt="image" src="https://user-images.githubusercontent.com/13292662/206096098-0a5d440b-278c-4cb5-b89a-bdf9b5bd7723.png">

Reply:
updated logic for this however still in the midst of fixing warning 
`Warning: Cannot update a component (`MoreDropdown`) while rendering a different component (`ForwardRef`).`

Reply:
fixed warning errors and moved dropdown links to a separate variable